source/quickstart.rst

```diff
@@ -6,22 +6,24 @@ Introduction
 
 Welcome to Manim DSL's documentation!
 
-Manim DSL is a Domain Specific Language that is made to help lecturers make animations for teaching data structures
-and algorithms
+Manim DSL is a domain specific language that helps educators create animations for teaching students about data structures
+and algorithms.
 
 Here is an example:
 
-.. code:: none
+.. code:: javascript
 
     let stack: Stack = new Stack;
     stack.push(5);
     stack.pop();
 
-Which would produce the video:
+This would produce the video:
 
+.. raw:: html
 
-<-- INSERT VIDEO -->
-
+    <div style="position: relative; padding-bottom: 56.25%; height: 0; overflow: hidden; max-width: 100%; height: auto;">
+        <iframe src="../../source/media/intro.mp4" frameborder="0" allowfullscreen style="position: absolute; top: 0; left: 0; width: 80%; height: 80%;"></iframe>
+    </div>
```

Comment:
Would this work when deployed? I'm not sure, but happy to merge and try it out

Reply:
Please address the above comment too don't merge yet

Reply:
Ah no, you're right - the directory is different in deployment to when you use `make html` locally. I'll try it using the command we use in GitHub Actions and then push again once I know it's correct

Reply:
Fixed now?

Reply:
Yep done!